### ISSUE TITLE
[icerec] Incremental Release V05-00-04 (Trac #1890)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1882]
This is a tracking ticket for icerec V05-00-04.

See comments for changes that need to be included.

Migrated from https://code.icecube.wisc.edu/ticket/1890
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:55", 
    "description": "This is a tracking ticket for icerec V05-00-04.\n\nSee comments for changes that need to be included.", 
    "reporter": "joertlin", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067175956889", 
    "component": "combo reconstruction", 
    "summary": "[icerec] Incremental Release V05-00-04", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-10-11T15:30:49", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20161011T15:32:54: joertlin commented:

> Low priority bug fix: http://code.icecube.wisc.edu/projects/icecube/changeset/150685/IceCube

**Comment by icecube-trac** [comment]
Trac update at 20161013T16:30:51: 
* olivas commented:

> In changeset 150752:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="150752"
> updated externals and release notes.  closes #1882.
> ```


* olivas changed resolution from "" to "fixed"
* olivas changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:55: 
* olivas commented:

> Milestone November 2016 deleted


* olivas changed milestone from "November 2016" to ""
